simplesamlphp/config/authsources.php

```diff
@@ -0,0 +1,339 @@
+<?php
+
+$config = [
+
+    // This is a authentication source which handles admin authentication.
+    'admin' => [
+        // The default is to use core:AdminPassword, but it can be replaced with
+        // any authentication source.
+
+        'core:AdminPassword',
```

Comment:
Is this overwritten somewhere?

Reply:
That first bit is the IDP to use when logging in to <site>/simplesaml.